packages/fxa-settings/src/pages/ResetPassword/en.ftl

```diff
@@ -0,0 +1,10 @@
+## ResetPassword page
+
+reset-password-heading = Reset password
+
+reset-password-warning-message = When you reset your password, you reset your account. You may lose some of your personal information (including history, bookmarks, and passwords). That’s because we encrypt your data with your password to protect your privacy. You’ll still keep any subscriptions you may have and Pocket data will not be affected.
```

Comment:
We should have Pocket added to brand names (if it's not already), and use the term reference her.